### DISCUSSION TITLE
Migrando para IA

### DISCUSSION THREAD

**Discussion by silvaMatheus** [frontendbr/forum#1223] [Discussão]
Pessoal alguém esta migrando para a área de IA ou pensa nessa possibilidade ?

**Comment by phsantiago** [comment]
Penso bastante mas é algo para longo prazo. Tenho juntado alguns materiais e estou bem curioso quanto a esse curso https://github.com/Avik-Jain/100-Days-Of-ML-Code

**Comment by rubenmarcus** [comment]
Ahahaha quando li esse tópico lembrei do video que uma empresa desenvolveu, do desenvolvedor desenhando na tela, e a IA criando componentes de react automáticamente. Logo pensei, em 5 anos acabou a profissão de front. Ahahaha mas sim, eu penso em IA, Data Science e Block Chain bastante

**Comment by didiraja** [comment]
Minhas possibilidades futuro a longo prazo são Data Science, Blockchain e brincar um pouquinho com Arduino em casa.
